### DISCUSSION TITLE
4k/7k Rank Distribution

### DISCUSSION THREAD

**Discussion by WimpyCursed** [ppy/osu#30785] [Q&A] [answered]
Hi! I couldn't find any details to this error(?)

It doesn't effect the game much, but when looking over mania rankings on profile pages, I notice that it hasn't been updated as global ranking has. 
![image_2024-11-19_113837340](https://github.com/user-attachments/assets/9416f52e-af77-41d1-9be2-1920b54f3e12)

I don't know if I am misunderstanding the rankings as a whole, but it doesn't appear right to me, especially considering what was displayed before the rework. Every inactive profile has their global ranking gone, but the same inactive profiles still have their 4k/7k ranks shown on their profile and on the leaderboards. 

Such as this profile
![image_2024-11-19_114342409](https://github.com/user-attachments/assets/ea2a068b-2d5a-4214-bd99-cbfb09c652c4)

Again, not an issue to the game itself, but it does confuse me as to why these rankings weren't updated.

**Comment by peppy** [comment, marked as answer]
Thanks for pointing this out. It is now updated.
